server/static/js/src/entry/blast/constants.js

```diff
@@ -0,0 +1,85 @@
+import * as validators from '../../utils/validators';
+
+export const DEFAULT_LEVEL = 'blastMonthly';
+
+export const BLAST_LEVELS = {
+  blastMonthly: {
+    bucket: 'standard',
+    bucketFull: 'Standard subscription',
+    installmentPeriod: 'monthly',
+    amount: '40',
+  },
+  blastYearly: {
+    bucket: 'standard',
+    bucketFull: 'Standard subscription',
+    installmentPeriod: 'yearly',
+    amount: '349',
+  },
+  blastTaxExempt: {
+    bucket: 'tax-exempt',
+    bucketFull: 'Tax-exempt subscription',
+    installmentPeriod: 'yearly',
+    amount: '325',
+  },
+  blastAcademicMonthly: {
+    bucket: 'academic',
+    bucketFull: 'Academic discount',
+    installmentPeriod: 'monthly',
+    amount: '20',
+  },
+  blastAcademicYearly: {
+    bucket: 'academic',
+    bucketFull: 'Academic discount',
+    installmentPeriod: 'yearly',
+    amount: '199',
+  },
+  blastLegislativeSession: {
+    bucket: 'legislative',
+    bucketFull: 'Legislative session-only',
+    installmentPeriod: 'session',
+    amount: '240',
+  },
+};
+
+export const BLAST_FORM_STATE = {
+  stripeEmail: {
+    value: '',
+    isValid: false,
+    validator: validators.isEmail,
+    message: 'Enter a valid email address',
+  },
+  first_name: {
+    value: '',
+    isValid: false,
+    validator: validators.isNotEmpty,
+    message: 'Enter your first name',
+  },
+  last_name: {
+    value: '',
+    isValid: false,
+    validator: validators.isNotEmpty,
+    message: 'Enter your last name',
+  },
+  pay_fees_value: {
+    value: 'False',
+    isValid: true,
+    validator: null,
+    message: null,
+  },
+  description: {
+    value: 'The Texas Tribune Circle Membership',
```

Comment:
**question:** Does this appear anywhere?

Reply:
Oops, that's a leftover. I'll remove.